vnext/Microsoft.ReactNative/Fabric/FabricUIManagerModule.cpp

```diff
@@ -0,0 +1,768 @@
+  }();
+
+  return providerRegistry;
+}
+
+void FabricUIManager::installFabricUIManager( // in android this is Binding::installFabricUIManager(
+                                              // jni::alias_ref<JRuntimeExecutor::javaobject> runtimeExecutorHolder,
+                                              // jni::alias_ref<jobject> javaUIManager,
+                                              // EventBeatManager *eventBeatManager,
+                                              // jni::alias_ref<JavaMessageQueueThread::javaobject>
+                                              // jsMessageQueueThread, ComponentFactory *componentsRegistry,
+                                              // jni::alias_ref<jobject> reactNativeConfig
+    ) noexcept {
+  // SystraceSection s("FabricUIManagerBinding::installFabricUIManager");
+
+  std::shared_ptr<const facebook::react::ReactNativeConfig> config =
+      std::make_shared<const ReactNativeConfigProperties>(m_context);
+
+  /*
+    enableFabricLogs_ =
+        config->getBool("react_fabric:enabled_android_fabric_logs");
+
+    if (enableFabricLogs_) {
+      LOG(WARNING) << "Binding::installFabricUIManager() was called (address: "
+                   << this << ").";
+    }
+    */
+
+  std::lock_guard<std::mutex> schedulerLock(m_schedulerMutex);
+
+  facebook::react::ContextContainer::Shared contextContainer = std::make_shared<facebook::react::ContextContainer>();
+  auto runtimeExecutor = SchedulerSettings::GetRuntimeExecutor(m_context.Properties());
+
+  // TODO: T31905686 Create synchronous Event Beat
+  facebook::react::EventBeat::Factory synchronousBeatFactory =
+      [/*eventBeatManager,*/ runtimeExecutor, localUIManager = weak_from_this(), context = m_context](
+          facebook::react::EventBeat::SharedOwnerBox const &ownerBox) {
+        return std::make_unique<AsyncEventBeat>(
+            ownerBox, /* eventBeatManager, */ context, runtimeExecutor, localUIManager);
+      };
+
+  facebook::react::EventBeat::Factory asynchronousBeatFactory =
+      [/*eventBeatManager,*/ runtimeExecutor, localUIManager = weak_from_this(), context = m_context](
+          facebook::react::EventBeat::SharedOwnerBox const &ownerBox) {
+        return std::make_unique<AsyncEventBeat>(
+            ownerBox, /* eventBeatManager, */ context, runtimeExecutor, localUIManager);
+      };
+
+  contextContainer->insert("ReactNativeConfig", config);
+
+  // Keep reference to config object and cache some feature flags here
+  m_reactNativeConfig = config;
+  m_collapseDeleteCreateMountingInstructions =
+      m_reactNativeConfig->getBool("react_fabric:enabled_collapse_delete_create_mounting_instructions") &&
+      !m_reactNativeConfig->getBool("react_fabric:enable_reparenting_detection_android") &&
+      !m_reactNativeConfig->getBool("react_fabric:enabled_layout_animations_android");
+
+  m_disablePreallocateViews = m_reactNativeConfig->getBool("react_fabric:disabled_view_preallocation_android");
```

Comment:
can you clarify what these *_android feature flags do? are they relevant in the context of RNW?